demos/satija-2020/src/convert_to_zarr.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import json
+from anndata import read_h5ad
+import pandas as pd
+from vitessce.data_utils import (
+    to_uint8,
+    sort_var_axis,
+    optimize_adata,
+)
+
+from constants import COLUMNS
+from generate_cell_sets import (
+    generate_leiden_cluster_cell_sets,
+    generate_cell_type_cell_sets
+)
+from utils import (
+    merge_cell_sets_trees
+)
+
+
+def generate_json_files(
```

Comment:
Good opportunity here to explain why we're doing this since this is really for others to see how the sausage is made i.e the limitation of `AnnData`.  As an aside, could probably store the JSON in the `uns` section, no?  I think `zarr` supports `JSON` encoding.

Reply:
Good point, I have added a comment about the motivation for using JSON within the function.